script.js

```diff
@@ -0,0 +1,65 @@
+console.log("test js on");
+// first task
+const dishes = [
+  "Spaghetti",
+  "Lasagne",
+  "Beans on toast",
+  "Porridge",
+  "Tomato soup",
+  "Salmon",
+];
+for (let i = 0; i < dishes.length; i++) {
+  console.log(dishes[i]); // dirty but ...
+}
+/* // 2nd task - Using JavaScript, create a button and add it to the html (the document)
+When the button is clicked:
+- Insert an h1 tag with the text “Javascript test”
+- Change the background color of the body */
+
+const body = document.body;
+const button = document.createElement("button");
+const tag = document.createElement("h1");
+function addTagChangeBg() {
+  tag.innerText = `Javascript test`;
+  body.append(tag);
+  body.style = `background-color: #CCC;`;
+  // function for click-event that adds h1 and changes background to gray
+}
+button.innerText = `BUTTON THAT DOES THINGS`;
+body.append(button);
+
+button.addEventListener("click", addTagChangeBg);
```

Comment:
This is a minor issue, but consider creating the "tag" variable inside the "addTagChangeBg" function, we want to create it only if the user clicks the button.